Author:  Emanuele Giaquinta
Fixes: http://bugs.irssi.org/index.php?do=details&task_id=841
Description: Remove OpenSSL SSL_OP_ALL. Patch from upstream.
Committer: Isis Lovecruft <isis@torproject.org>
Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=667014

--- a/src/core/network-openssl.c
+++ b/src/core/network-openssl.c
@@ -406,6 +406,7 @@
                g_error("Could not allocate memory for SSL context");
                return NULL;
        }
+       SSL_CTX_set_options(ctx, SSL_OP_NO_SSLv2);
 
        if (mycert && *mycert) {
                char *scert = NULL, *spkey = NULL;
